### ISSUE TITLE
Training Code

### ISSUE THREAD

**Issue by canuck6ix** [zhangboshen/A2J#16]
Thanks for sharing your inference code. I was wondering if your [issues previously mentioned](https://github.com/zhangboshen/A2J/issues/7) are resolved now so you can share your training code as well, thanks!

**Comment by zhangboshen** [comment]
Hi, we will considering that release our training scripts in the near future, but I cannot be sure exactly when, sorry.

**Comment by canuck6ix** [comment]
Fair enough, thanks. 
Meanwhile could you please clarify the following regarding data augmentation:
- in-plane rotation range
- in-plane and depth scaling ranges
- Is Gaussian noise added to the depth map, and what are its parameters? 
- By `0.5` probability you mean Gaussian noise is added to 50% of the depth map, or 50% of training data?

Regarding loss function:
- you use `lambda = 3` for balancing loss. Is it because loss1 and loss2 are of different scale? Meaning loss1 is one third of loss2, hence multiplying by 3? Or you intend to emphasize joint loss compared to anchor surrounding loss?

**Comment by happyCodingSusan** [comment]
Thank you for the good work! Expect you open the training source as well!

**Comment by zhangboshen** [comment]
Hi, we just released our training code: https://github.com/zhangboshen/A2J/tree/master/src_train
